includes/heimdalCurl.inc

```diff
@@ -18,11 +18,32 @@ class HeimdalCurl extends MicroCURL {
     * @return string
     */
     public function getUserInfo($bearer, $config) {
-        $this->set_url($config['heimdal_client_url'] . 'userinfo/');
-        $this->set_option(CURLOPT_HTTPHEADER, array("Authorization: Bearer $bearer"));
-        $attributes = $this->get();
+      $this->set_url($config['heimdal_client_url'] . 'userinfo/');
+      $this->set_option(CURLOPT_HTTPHEADER, array("Authorization: Bearer $bearer"));
 
-        return $attributes;
+      $attributes = $this->get();
+
+      /* THIS IS TEMPORARY UNTIL HEIMDAL HAS REPLACED FORS WITH IDP */
+      $decoded_att = json_decode($attributes);
+
+      if (isset($decoded_att->attributes)) {
+        if (!isset($decoded_att->attributes->dbcidp)) {
+          // DBCIDP is not present, we need to call IDP to get the rights
```

Comment:
Mener du FORS her?

Reply:
Nej, hvis dbcidp ikke er til stede skal vi kalde IDP for at få oplysningerne. Jeg går med livrem og seler her for at være sikker på at IDP bliver hentet.

Reply:
ok, det forvirrer mig nok bare at der står forsrights nedenunder :)

Reply:
Det nedenunder er oprydning, jeg fjerner forsrights tingene hvis de findes og indsætter idp i stedet for.